app/controllers/items_controller.rb

```diff
@@ -693,16 +692,16 @@ def save_and_reindex
   # Permissions
 
   # check that the user can carry out this item modification
-  def forbid_modify
-    return true if current_user.is_admin || @object.can_manage_content?(current_user.roles(@apo))
-    render :status => :forbidden, :text => 'forbidden'
-    false
+  def authorize_manage_obj!
+    authorize! :manage_item, @object
   end
```

Comment:
We did discuss the defaults with regard to `manage_content` vs. `manage_item` and resolved the latter is preferred.  I could be important that the prior logic only includes `is_admin` and not `is_manager`.